src/SimpleBoundaryValueDiffEq.jl

```diff
@@ -1,5 +1,19 @@
 module SimpleBoundaryValueDiffEq
 
-# Write your package code here.
+using DiffEqBase, DifferentialEquations, NonlinearSolve, Plots
```

Comment:
As for the nonlinear solving in simple shooting methods, I think SimpleNonlinearSolve.jl is enough?

Reply:
Yes we should just use simplenonlinearsolve